mcs/class/System/System.Net.NetworkInformation/NetworkInterface.cs

```diff
@@ -698,11 +706,10 @@ public override IPv4InterfaceStatistics GetIPv4Statistics ()
 					// interface is up or down. There is a way to get more detailed information but
 					// it requires an instance of the Android Context class which is not available
 					// to us here.
-					bool is_up = false;
-					if (_monodroid_get_network_interface_up_state (Name, ref is_up))
-						return is_up ? OperationalStatus.Up : OperationalStatus.Down;
-					else
+					bool is_up;
```

Comment:
We trust native code to initialize? Or locals are zeroed anyway?

Reply:
Locals are *not* zero'd out, afaik.